pmm-app/src/shared/components/Elements/Table/Table.styles.ts

```diff
@@ -66,5 +66,14 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => {
     checkboxColumn: css`
       width: 20px;
     `,
+    tableClassicExplain: css`
+    table {
+      td {
+        white-space: normal;
```

Comment:
why do we need this? why not `word-break: break-all;`

Reply:
We do not need it really, just tried to make clear that this property could be used.
word-break: break-word could be useful to keep words as far but break-all works well in any case.
Fixed.